discovery-infra/test_infra/utils.py

```diff
@@ -348,6 +348,30 @@ def is_cluster_in_status(client, cluster_id, statuses):
         log.exception("Failed to get cluster %s info", cluster_id)
 
 
+def get_cluster_validation_value(cluster_info, validation_section, validation_id):
```

Comment:
i don't think this is the place. It is not utils functions, lets move them to cluster object